modular_zubbers/code/modules/research/designs/misc_designs.dm

```diff
@@ -7,3 +7,39 @@
 	build_path = /obj/item/disk/plantgene
 	category = list("Electronics")
 	departmental_flags = DEPARTMENT_BITFLAG_SERVICE
+
+/datum/design/nitrogen_tank
+	name = "Nitrogen Tank"
+	desc = "An empty nitrogen tank."
+	id = "nitrogen_tank"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron =SHEET_MATERIAL_AMOUNT)
+	build_path = /obj/item/tank/internals/nitrogen
+	category = list(
+		RND_CATEGORY_EQUIPMENT + RND_SUBCATEGORY_EQUIPMENT_GAS_TANKS
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_CARGO
+
+/datum/design/nitrogen_tank_belt
+	name = "Nitrogen Internals Tank"
+	desc = "An empty nitrogen tank designed specifically for use as internals."
+	id = "nitrogen_tank_belt"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron =SHEET_MATERIAL_AMOUNT)
+	build_path = /obj/item/tank/internals/nitrogen/belt
```

Comment:
Need to keep in mind we're giving the entire station access to 24 volume tanks, whereas emergency tanks have a volume of 3

Reply:
This was supposed to already have been removed - it's gone on my local end. How are you even seeing this part